assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -13,7 +17,77 @@ import './style.scss';
  */
 const Block = () => {
 	return (
-		<Placeholder>Checkout block coming soon to store near you</Placeholder>
+		<CheckoutForm>
+			<FormStep
+				className="wc-blocks-checkout__billing-fields"
+				title={ __(
+					'Contact information',
+					'woo-gutenberg-products-block'
+				) }
+				description={ __(
+					'We’ll use this email to send you details and updates about your order.',
+					'woo-gutenberg-products-block'
+				) }
+				stepNumber={ 1 }
+				id="checkout-billing-fields"
+				secondaryAction={
+					<Fragment>
+						{ __(
+							'Already have an account? ',
+							'woo-gutenberg-products-block'
+						) }
+						<a href="/wp-login.php">
+							{ __( 'Log in.', 'woo-gutenberg-products-block' ) }
+						</a>
+					</Fragment>
+				}
+			>
+				<Placeholder>A checkout step, coming soon near you</Placeholder>
+			</FormStep>
+			<FormStep
+				className="wc-blocks-checkout__shipping-fields"
+				title={ __(
+					'Shipping address',
+					'woo-gutenberg-products-block'
+				) }
+				description={ __(
+					'Enter the physical address where you want us to deliver your order.',
+					'woo-gutenberg-products-block'
+				) }
+				stepNumber={ 2 }
+				id="checkout-shipping-fields"
```

Comment:
Do we need the `checkout-` prefix? In case these ids are there to make them link-able, it would be better `#shipping-fields` than `#checkout-shipping-fields`, no?
```suggestion
				id="shipping-fields"
```
And a nitpicking comment :nerd_face: I feel it quite weird that `id` is the last attribute, I'm used to seeing it as the first one (or seeing them sorted alphabetically).